packages/react-server/src/server/test/floopydoop.test.tsx

```diff
@@ -18,21 +17,20 @@ jest.mock('@shopify/sewing-kit-koa', () => ({
 }));
 
 describe('createServer()', () => {
-  afterAll(() => {
-    rack.unmountAll();
-  });
+  afterAll(unsaddle);
 
   it('starts a server that responds with markup', async () => {
     function MockApp() {
       return <div>markup</div>;
     }
 
-    const wrapper = await rack.mount(({ip, port}) =>
-      createServer({port, ip, render: () => <MockApp />}),
+    const wrapper = await saddle((port, host) =>
```

Comment:
Here we use the factory version of `saddle` because `createServer` doesn't follow the usual `Listenable` interface most node server libs do